### ISSUE TITLE
Bug #15 - Remove the "Player" right navbar button from all VCs when an episode or clip finishes playing

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse#125]
Fabric [crash #15](https://fabric.io/podverse/ios/apps/fm.podverse.podverse/issues/56b8fbb6f5d3a7f76bae21bb) would be fixed if this feature is implemented. 

I believe this can be done with some kind of notification. Basically if I'm on the PodcastTableVC (or any other VC), and an episode finishes playing, then that episode will be automatically deleted, but the "Player" button in the upper right hand corner persists. If you press the "Player" button, the app will crash because there is no longer a valid episode loaded in the MediaPlayer.

What I want to happen is, if you are on a VC and the episode finishes playing, the "Player" button is removed as soon as the episode is deleted.

I think this can be done with a notification of some kind, but I'm not sure...

**Comment by mitchdowney** [comment]
Ah I see I can do this with NSNotificationCenter and adding observers.

**Comment by mitchdowney** [comment]
This has been pushed to branch fixBug#15. Seems to be working according to my tests...

**Comment by mitchdowney** [comment]
This bug happened again today. I think it was because an episode finished while my phone was locked, then I opened the app, the MediaPlayerVC loaded even though it's nowPlayingEpisode was deleted, and then somewhere down the MediaPlayerVC loading process a nil value was unwrapped.

I added a check in the viewDidLoad that will nav back out of the MediaPlayerVC is no nowPlayingEpisode or nowPlayingClip exists.
